ProfessorAssessor/View Controllers/SignUpViewController.h

```diff
@@ -8,6 +8,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (strong, nonatomic) IBOutlet UILabel *schoolSelection;
 @property (strong, nonatomic) IBOutlet UITextField *username;
 @property (strong, nonatomic) IBOutlet UITextField *password;
+@property (strong, nonatomic) NSDictionary *requestResult;
```

Comment:
I imagine you'll delete this based on other comments but for learning sake, this needs to be more descriptive. Request what? Resulting in what? What kind of dictionary is this?